index.js

```diff
@@ -19,17 +19,25 @@ module.exports.env = function() {
 };
 
 module.exports.os = function() {
-  if (typeof os.userInfo === 'function') {
-    return os.userInfo().username;
-  }
+  try {
+    if (typeof os.userInfo === 'function') {
+      return os.userInfo().username;
+    }
+  } catch (_) {}
```

Comment:
Can you share the actual error and error code thrown? I would prefer to only catch errors explicitly, rather then broadly. I find that  doing so prevents bit rot.

```
catch(e) {
  if (e !== null && typeof e === 'object' && e.code === <you tell me>) {
   // we understand what happened, and we know its safe to recover
  } else {
   // we don't know what happened, so we should need to raise as it may not be safe to continue.
   // we also do this, to ensure future exceptional issues are surfaced so they can be fixed.
   throw e;
  }
}
```

Reply:
Thanks for the quick response and feedback. It's specifically an ENOENT (no file or dir) when calling the os module from inside of Docker that we're seeing. I'll push an update shortly.